docker-compose.yaml

```diff
@@ -0,0 +1,27 @@
+---
+version: "3.8"
+services:
+  bob-gui:
+    build:
+      context: ./
+      dockerfile: dockerfiles/alpine/Dockerfile
+    networks:
+      bobnet:
+        ipv4_address: 192.168.17.11
+    ports:
+      - "9000:9000"
+  bob-gui-dev:
```

Comment:
Why are here the 2 containers with the application running? If this file for dev purposes only, then only `dev` version should be used. If it is for external users, then `release` version should be used. Not both at the same time

Reply:
I think for external users this file is not needed. For them it is better to create a separate compose file in the future, that will use docker image from docker hub